src/main/webapp/app/course/manage/detail/course-detail-doughnut-chart.component.html

```diff
@@ -1,16 +1,16 @@
 <div class="doughnut mx-3">
     <a *ngIf="titleLink" [routerLink]="['/course-management', course.id, titleLink]">
-        <h5 class="text-center">{{ 'artemisApp.course.detail.' + doughnutChartTitle + 'Title' | artemisTranslate }}</h5>
+        <div class="text-center h5">{{ 'artemisApp.course.detail.' + doughnutChartTitle + 'Title' | artemisTranslate }}</div>
```

Comment:
The headline structure is currently completely broken. Since the whole page will probably be refactored in the next months, I did not completely fix it. But I remove some of the HTML headline tags of some text, where I think, that users dont expect it to be a headline.
Imo those texts are more of something between a figure description and a nav link, than a headline.

But yes, since we dont expect this page to be crawled, the advantage (beside best-practice) of a maintained headline structure are better working screen readers.

Reply:
I wanted to refer to `<div class="h4">...</div>` vs `<h4>...</h4>`

On the Headlines itself, I do not have a strong opinion right now - looked fine; though I think we should come up with some guidelines to unify it in Artemis in the future, so we might have to rework it in the near future anyways 😅